### ISSUE TITLE
[Usability]A11y_PowerShellEditorServices_Terminal_Usability : After spiliting Terminal panel, When we move to next split terminal Screenreader does not read any instuction.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [microsoft/vscode#62029]
Environment Details:
VSCode Version : 1.28.2
OS Version : Win10

Additional information : Usability issue 

Repro Steps:
1.Launch Visual Studio code desktop application.
2.Navigate to "View" item of menu bar and select "Integrated Terminal"->Navigate to "Terminal" tab and select "PowerShell Integrated Console" from combo box.
3.Navigate "Terminal panel" -->Navigate to "Split terminal button" & split terminal into 2,3 screens.
4.Start NVDA & check whether it is reading proper instruction or not.
 
Actual:
After Splitting Terminal panel, When we move to next split terminal Screenreader does not read any instruction that at which terminal we are currently present.
 
Expected:
After Splitting Terminal panel, When we move to next split terminal Screenreader should read the instruction that at which terminal we are currently present.
 
User Impact:
Screen reader users finds difficult to understand the functionality as well unable to identify at which terminal panel he/she editing.

Does this issue occur when all extensions are disabled?: Yes

Attachment:
[A11y_PowerShellEditorServices_Terminal_Usability1.pptx](https://github.com/Microsoft/vscode/files/2524921/A11y_PowerShellEditorServices_Terminal_Usability1.pptx)

**Comment by isidorn** [comment]
Closing this as this works just fine for me now. When I open a new terminal focus moves to the terminal and we nicely read the aria label of the terminal.
